volmaster/daemon.go

```diff
@@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
 	r := mux.NewRouter()
 
 	postRouter := map[string]func(http.ResponseWriter, *http.Request){
-		"/request":      d.handleRequest,
-		"/create":       d.handleCreate,
-		"/copy":         d.handleCopy,
-		"/mount":        d.handleMount,
-		"/mount-report": d.handleMountReport,
-		"/unmount":      d.handleUnmount,
-		"/remove":       d.handleRemove,
+		"/global":                           d.handleGlobalUpload,
+		"/volumes/create":                   d.handleCreate,
```

Comment:
ah, I see, this should be fine as well.

I was just wondering wrt consistency of the endpoints. Right now, in policy and snapshot endpoints we are passing the resource name in the URL but for volumes we don't, so thought I will check.